src/Domain/DomainHelpers.cpp

```diff
@@ -54,23 +54,23 @@ std::vector<size_t> get_common_global_corners(
 // in block2, this function first repackages them into arrays and
 // then makes a std::pair of the arrays.
 template <size_t VolumeDim>
-std::pair<std::array<Direction<VolumeDim>, VolumeDim>,
-          std::array<Direction<VolumeDim>, VolumeDim>>
+std::pair<std::array<domain::Direction<VolumeDim>, VolumeDim>,
```

Comment:
There seems to be a lot of unnecessary change going on in this file.  I think you only need to change four lines:
```patch
diff --git a/src/Domain/DomainHelpers.cpp b/src/Domain/DomainHelpers.cpp
index 56571316a..2b2800cd5 100644
--- a/src/Domain/DomainHelpers.cpp
+++ b/src/Domain/DomainHelpers.cpp
@@ -31,6 +31,7 @@
 #include "Utilities/GenerateInstantiations.hpp"
 #include "Utilities/Gsl.hpp"
 
+namespace domain {
 namespace {
 
 // returns the common corners in the global corner numbering scheme.
@@ -405,9 +406,9 @@ void set_identified_boundaries(
     const auto& face2 = pair.second;
     ASSERT(face1.size() == face2.size(),
            "Each set must have the same number of corners.");
-    size_t id1 = ::find_block_id_of_external_face<VolumeDim>(
+    size_t id1 = find_block_id_of_external_face<VolumeDim>(
         face1, corners_of_all_blocks);
-    size_t id2 = ::find_block_id_of_external_face<VolumeDim>(
+    size_t id2 = find_block_id_of_external_face<VolumeDim>(
         face2, corners_of_all_blocks);
     const auto face1_canon =
         get_common_local_corners<VolumeDim>(corners_of_all_blocks[id1], face1);
@@ -1055,4 +1056,5 @@ GENERATE_INSTANTIATIONS(INSTANTIATE, (1, 2, 3), (Frame::Grid, Frame::Inertial))
 #undef DIM
 #undef DTYPE
 #undef INSTANTIATE
+}  // namespace domain
 /// \endcond
```